site/content/docs/user/configuration.md

```diff
@@ -289,6 +289,11 @@ included in the Preferences -> Resources -> File Sharing.
 
 For more information see the [Docker file sharing guide.](https://docs.docker.com/docker-for-mac/#file-sharing)
 
+**NOTE**: Mount propagation is not possible with Mac or Windows Docker Desktop,
+due to limitation of Docker Desktop implementation. 
```

Comment:
I'm not sure how to tersely cover this off the top of my head but, this is not quite right.

You cannot use mount propagation *when the mount is to a path on the mac or windows host* *when using docker desktop* (and presumably other tools, but I can't say for certain, I don't other VM <> host boundaries can handle this either).

When using docker desktop you could still use this setting *if* the hostPath is to a location on the VM and not the mac/windows host. Which is generally unusual I think, but possible.

If #1966 lands, it should also be possible for that, because volumes are generally contained within the VM disk.

Reply:
So basically ... mount propagation doesn't work across the macOS / Windows | Docker Desktop Virtual Machine boundary. Mounts that cross that boundary can't be expected to support this, and that limitation comes from Docker.